pkg/occlient/templates.go

```diff
@@ -419,3 +419,7 @@ func addBootstrapVolumeMount(dc *appsv1.DeploymentConfig, dcName string) {
 		})
 	}
 }
+
+func getComponentContainer(dc *appsv1.DeploymentConfig) corev1.Container {
```

Comment:
do we really need this function? It doesn't do much

Reply:
yeah but i was just thinking if and when we support multi-container component, it might be single point of update if we have this func

Reply:
we already have `Containers[0]` everywhere, I would rather stick to it for now. It will make it easier to find places where we need to update this. Right now this function just complicates things

Reply:
ok I'll quickly fix it